js/main.js

```diff
@@ -67,6 +71,16 @@ if (mapSection && adForm) {
     changeFieldsValue(getLatLngRoundedString(mainMarker.getLatLng()), adFormAddress);
   }
 
+  const checkFilters = (realtyObject) => {  //ф-ия проверки объекта фильтрами
+    return (
+      isPropertyFitsFilter(realtyObject.offer.type, filterHousingType, DEFAULT_FILTER_VALUE)
+    );
+  }
```

Comment:
```suggestion
  const checkFilters = (realtyObject) =>  isPropertyFitsFilter(realtyObject.offer.type, filterHousingType, DEFAULT_FILTER_VALUE)
```

Лишних скобок можно избежать вовсе